src/main/java/entity/Match.java

```diff
@@ -0,0 +1,25 @@
+package entity;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.FieldDefaults;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class Match {
+    int id;
+    int tournamentId;
+    int firstTeamId;
+    int secondTeamId;
+    int round;
+    String matchResult;
+
+    public String toString(){
+        return "id:" + id + " tourID:" + tournamentId + " firstTeam:" + firstTeamId +
```

Comment:
In case of complex patterns I prefer to use `java.lang.String#format(java.lang.String, java.lang.Object...)`